cadl-tests/src/test/java/com/dictionary/DictionaryTests.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.dictionary;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DictionaryTests {
+
+    private final BooleanValueClient client = new BooleanValueClientBuilder().buildClient();
+    private final Map<String, Boolean> expected = new HashMap<>();
+    {
+        expected.put("k1", true);
+        expected.put("k2", false);
+    }
+
+    @Test
+    public void testBooleanGet() {
+        Map<String, Boolean> response = client.get();
+        Assertions.assertEquals(expected, response);
+    }
+
+    @Test
+    public void testBooleanPut() {
+        client.put(expected);
+    }
```

Comment:
@haolingdong-msft Added 1 test to verify return type as generic works.

Feel free to add others.